torch/nn/quantized/dynamic/modules/rnn.py

```diff
@@ -285,6 +285,32 @@ def quantize_and_pack(w, b):
         return qRNNBase
 
 
+    def _weight_bias(self):
+        # Returns a dict of weights and biases
+        weight_bias_dict = {'weight' : {}, 'bias' : {}}
+        count = 0
+        num_directions = 2 if self.bidirectional else 1
+        for layer in range(self.num_layers):
+            for direction in range(num_directions):
+                suffix = '_reverse' if direction == 1 else ''
+                key_name1 = 'weight_ih_l{layer_idx}{suffix}'.format(layer_idx=layer, suffix=suffix)
+                key_name2 = 'weight_hh_l{layer_idx}{suffix}'.format(layer_idx=layer, suffix=suffix)
+                packed_weight_bias = self._all_weight_values[count].param.__getstate__()[0][4]
```

Comment:
Can you add some comments here explaining the choice of the indexes? Is this accessing the tuple `CellParamsSerializationType`?